src/commands/set.json

```
@@ -85,6 +89,21 @@
                 "name": "value",
                 "type": "string"
             },
+            {
+                "name": "comparison-value",
+                "type": "string",
+                "token": "IFEQ",
+                "since": "8.2.0",
```

```suggestion
                "since": "8.1.0",
```

and move this block to within the "oneOf" for NX | XX below. The `"since": "2.6.12"` can be moved from the oneOf container to the NX and XX args individually.

---

I also like this approach. Doesn't make a lot of sense to combine IFEQ with NX | XX.

---

addressed!